llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -921,6 +950,46 @@ static std::string getESIMDIntrinSuffix(id::FunctionEncoding *FE,
   return Suff;
 }
 
+static void translateBlockLoad(CallInst &CI, bool IsSLM) {
+  IRBuilder<> Builder(&CI);
+
+  APInt Val =
+      parseTemplateArg(CI, 2, ESIMDIntrinDesc::GenXArgConversion::TO_I64);
```

Comment:
could we make the `2` into a variable to explain what it is for clarity?

like `int AlignmentTemplateArgIdx = 2;`

Reply:
Fixed in 1e722d1af21d4d979f9e352e053a0ae4d340e45a